test/integration-tests/Seaport.test.ts

```diff
@@ -21,125 +19,16 @@ import { getPermitSignature } from './shared/protocolHelpers/permit2'
 import { abi as TOKEN_ABI } from '../../artifacts/solmate/src/tokens/ERC20.sol/ERC20.json'
 const { ethers } = hre
 
-describe('Seaport v1.1', () => {
+describe('Seaport v1.5', () => {
   let router: UniversalRouter
   let permit2: Permit2
   let planner: RoutePlanner
-  let cryptoCovens: ERC721
-  let weth: Contract
-  let bob: SignerWithAddress
+  let townStarNFT: ERC1155
   let alice: SignerWithAddress
 
-  // @dev We have to separate the ERC20 tests because we use orders taken from a much
-  // newer block. If we were to replace all of the example orders with newer ones,
-  // many existing tests would break.
-  describe('ERC20 -> NFT', () => {
```

Comment:
wait why is this getting taken out again?

Reply:
just bc i didnt have enough sample test data yet, but will make an issue so we can add ERC20 tests. v1.4 is tested with ERC20s and functions the same though :)